README.md

```diff
@@ -0,0 +1,31 @@
+# Checking BCA Eligibility
+
+Write a function to decide whether or not an application is able to be automatically
+accepted for a Business Cash Advance (BCA). The application must conform to the following criteria in order to be eligible:
+
+* The amount requested is between 5,000 and 50,000 inclusive
+* The transaction average in each month must exceed the requested amount
+* The business must have been operating for more than 12 months
+* If there are no transactions in a month for the last 12 months, then the assumed
+  transaction value for that month is the average of all other months that do have transactions.
+* If the amount requested is above 25,000, then the transaction history must exceed 12 months, with no months

```

Comment:
`must exceed 12 months`. How many must it go up to?

Reply:
Confused about `with no months having no transactions`. Should they be filled in with the average as stated in the above bullet point?